lib/bridge/PublicitySyncer.js

```diff
@@ -181,8 +181,12 @@ PublicitySyncer.prototype._solveVisibility = Promise.coroutine(function*() {
         let currentState = this._visibilityMap.roomVisibilities[roomId];
         let correctState = shouldBePrivate(roomId, []) ? 'private' : 'public';
 
+        // Use the server domain of the first mapping
+        // 'localhost #channel1' => 'localhost'
+        const networkId = this._visibilityMap.mappings[roomId][0].split(' ')[0];
```

Comment:
Is `this._visibilityMap.mappings[roomId]` **guaranteed** to be non-null? We seem to do a null guard on `:140` but we don't here. But `:127` makes me think that yes, it is guaranteed to be non-null. But then why the null guard on `:140`? Some clarification here would be good.